lib/rest/axios.js

```diff
@@ -188,6 +189,7 @@ export function Axios(config) {
           if (method === 'GET' && options.data !== '') {
             let query = '?' + queryString.stringify(params);
             options.url += query;
+            delete options.data
```

Comment:
This is being done only when `isVoiceReq` is true. Need to do same in `else { ... }` block. Also this might allow data to be there if it's just empty body `""` and it's a `GET` request. Just check if for all `isVoiceReq` there's no `GET` call with empty body data `""`.

Reply:
Makes sense, thanks